table.csv

```diff
@@ -483,6 +483,12 @@ ssz,                            serialization,  0xb501,         draft,     Simpl
 ssz-sha2-256-bmt,               multihash,      0xb502,         draft,     SSZ Merkle tree root using SHA2-256 as the hashing function and SSZ serialization for the block binary
 iscc,                           softhash,       0xcc01,         draft,     ISCC (International Standard Content Code) - similarity preserving hash
 zeroxcert-imprint-256,          zeroxcert,      0xce11,         draft,     0xcert Asset Imprint (root hash)
+varsig,                         varsig,         0xd000,         draft,     Namespace for all not yet standard signature algorithms
+es256k,                         varsig,         0xd0e7,         draft,     ES256K Siganture Algorithm (secp256k1)
+bls-12381-g1-sig,               varsig,         0xd0ea,         draft,     G1 signature for BLS-12381-G2
+bls-12381-g2-sig,               varsig,         0xd0eb,         draft,     G2 signature for BLS-12381-G1
+eddsa,                          varsig,         0xd0ed,         draft,     Edwards-Curve Digital Signature Algorithm
+eip-191,                        varsig,         0xd191,         draft,     EIP-191 Ethereum Signed Data Standard
```

Comment:
@rvagg This one is an exception to the above comment, since it uses secp256k1 but with a "nonstandard" (non-ECDSA) signature method.